extensions/locals.tf

```diff
@@ -1,6 +1,16 @@
 locals {
   plans = yamldecode(file("${path.module}/plans.yml"))
 
+  extensions_for_environment = {
+    for k, v in var.args.services :
+    k => merge(v, {
+      environments = {
+        for ek, ev in v["environments"] :
+        ek => ev if contains(["*", var.environment], ek)
+      }
+    })
+  }
```

Comment:
Kudos to @acodeninja for this fix.

Reply:
should we have a small comment to describe what is happening here?